maps/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.conf.urls import url                                                                                                                              
+from . import views
+
+urlpatterns = [ 
+    url(r'^maps/', views.default_map, name="maps"),
+]
```

Comment:
Minor change but looks like git wants a new line at the end of this file, which is why the red circle with the minus sign is there